test/demo/contextListener.test.ts

```diff
@@ -0,0 +1,46 @@
+import 'jest';
+import * as fdc3Remote from './utils/fdc3RemoteExecution';
+import {getFinConnection} from './utils/fin';
+
+
+describe('advanced puppeteer functionality', () => {
+    const testManagerIdentity = {uuid: 'test-app', name: 'test-app'};
+    const testIdentity1 = {uuid: 'test-app-1', name: 'test-app-1'};
+    const testIdentity2 = {uuid: 'test-app-2', name: 'test-app-2'};
+
+    it('can can register and trigger a context listener', async () => {
```

Comment:
Weird question, but what does `it` mean to you? Like, in your head are you saying "it", or "I.T". I don't get this Jest's choice of abbreviation 🤷‍♂️

(And on the other tests you're stating the `it` and `describe` messages with capital letters)

Reply:
Defnitely some grammatical inconsistencies on my part. Lets discuss offline as a team and decide on some standard.

Reply:
Yep. Another thing to take into account is how Jenkins interprets these strings in the results xml file - it treats '.' as a package separator, so that's why the hierarchy when looking at unit test results in Jenkins is a bit odd - there's the odd test that has a '.' in its name

Reply:
Ah, ok. I hadn't noticed that. Definitely needs fixing.